src/librustdoc/html/static/js/settings.js

```diff
@@ -153,12 +141,13 @@
                 // This is a checkbox toggle.
                 const checked = setting["default"] === true ? " checked" : "";
                 output += `\
-<label class="settings-toggle">\
+<div class="setting-line">\
+<label class="setting-check">\
```

Comment:
While we're at it, please keep the correct indent level since you added a new level.